pkg/debugger/namespace_test.go

```diff
@@ -3,22 +3,33 @@ package debugger
 import (
 	"net/http/httptest"
 	"testing"
+	"time"
 
 	"github.com/golang/mock/gomock"
 	tassert "github.com/stretchr/testify/assert"
 
 	"github.com/openservicemesh/osm/pkg/catalog"
+	tresorFake "github.com/openservicemesh/osm/pkg/certificate/providers/tresor/fake"
+	"github.com/openservicemesh/osm/pkg/compute"
+	"github.com/openservicemesh/osm/pkg/messaging"
 	"github.com/openservicemesh/osm/pkg/tests"
 )
 
 // Tests getMonitoredNamespaces through HTTP handler returns a the list of monitored namespaces
 func TestMonitoredNamespaceHandler(t *testing.T) {
 	assert := tassert.New(t)
 
-	mock := catalog.NewMockMeshCataloger(gomock.NewController(t))
+	mock := compute.NewMockInterface(gomock.NewController(t))
+	stop := make(chan struct{})
+	meshCatalog := catalog.NewMeshCatalog(
+		mock,
+		tresorFake.NewFake(time.Hour),
+		stop,
+		messaging.NewBroker(stop),
+	)
 
 	ds := DebugConfig{
-		meshCatalog: mock,
+		meshCatalog: meshCatalog,
```

Comment:
Just a note, there will be some conflicts in pkg/debugger if #5225 is merged before this PR.